emacs-travis.mk

```diff
@@ -50,6 +50,10 @@ EMACSCONFFLAGS += --with-modules
 endif
 endif
 
+ifeq ($(TRAVIS_OS_NAME),osx)
+EMACSCONFFLAGS += --with-gameuser=$(USER)
```

Comment:
Please add a comment as to why we pass that here. The line from the commit message is fine.